cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -35,7 +34,8 @@ def as_tuple(self) -> tuple[str, str]:
         return self.space, self.external_id
 
 
-class PropertyType(ABC):
+@dataclass
+class PropertyType:
```

Comment:
Why not abstract anymore?

Reply:
It lacks any abstract methods, so it can be instantiated. Also, `mypy` complained about `output = asdict(self)` (I'll see if I can make this work without a type ignore comment)

Reply:
```
cognite/client/data_classes/data_modeling/data_types.py:43: error: No overload
variant of "asdict" matches argument type "PropertyType"  [call-overload]
            output = asdict(self)
```

Reply:
The issue magically vanished now 🥷